src/Redux/userSlice.js

```diff
@@ -1,22 +1,44 @@
-//this does nothing at the moment I just set it up because
```

Comment:
this sets up the methods we will be using to store the logged in user in our Redux state